build/build_compile_image.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+KMESH_DIR=$(dirname $(pwd))
+
+docker buildx build --platform linux/amd64 -t kmesh:build -f docker/kmesh-compiler.dockerfile $KMESH_DIR
```

Comment:
replace `kmesh:build` with ghcr.io/kmesh.net/build-tool:{xxxtag}

Reply:
> replace `kmesh:build` with ghcr.io/kmesh.net/build-tool:{xxxtag}

This will be changed later and does not affect the current pr